mirrord/agent/src/entrypoint.rs

```diff
@@ -857,12 +946,9 @@ pub async fn main() -> AgentResult<()> {
     );
 
     let args = cli::parse_args();
+    let second_process = matches!(std::env::var(SECOND_PROCESS_ENV), Ok(value) if value == "true");
```

Comment:
```suggestion
    let second_process = std::env::var(SECOND_PROCESS_ENV).is_ok();
```
Can be simpler, I think